concord-consortium/codap
# Update to latest CFM with V2 DocumentServer API

Update to latest CFM
- includes PR#45
- SHA 5482fc085be68a8255e527db7d2e18f2df89bb6d

Support latest CFM changes
- Enable/configure LaraProvider
- Enable Concord Cloud deprecation phase I
- Eliminate "Authorize" button from user entry dialog
- Eliminate URL processing now handled by CFM

Update deploy script to use new internal server name
- codap.concord.org => codap-server.concord.org

Note: Deployed to http://codap.concord.org/~kswenson/v2docstore for testing. Should not be merged until testing indicates it's sufficiently stable to merge to master.
